torchvision/datasets/ucf101.py

```diff
@@ -42,10 +44,23 @@ class UCF101(VisionDataset):
             - label (int): class of the video clip
     """
 
-    def __init__(self, root, annotation_path, frames_per_clip, step_between_clips=1,
-                 frame_rate=None, fold=1, train=True, transform=None,
-                 _precomputed_metadata=None, num_workers=1, _video_width=0,
-                 _video_height=0, _video_min_dimension=0, _audio_samples=0):
+    def __init__(
+        self,
+        root: str,
+        annotation_path: str,
+        frames_per_clip: int,
+        step_between_clips: int = 1,
+        frame_rate: Optional[int] = None,
+        fold: int = 1,
+        train: bool = True,
+        transform: Optional[Callable] = None,
+        _precomputed_metadata: Optional[Dict[str, Any]] = None,
+        num_workers: int = 1,
+        _video_width: int = 0,
+        _video_height: int = 0,
+        _video_min_dimension: int = 0,
+        _audio_samples: int = 0
+    ) -> None:
```

Comment:
I'm just curious here: were you using something (an IDE, a type checker (mypy??) or something else) that was prompting you to type the return of `__init__`?

If yes, I'm curious to know which one, and why they would do that. But if not, I would recommend not to type those in future PRs, as this is essentially useless.

I have the same question/remark for typing the return type of `__len__` below actually

Reply:
Actually, I don't since I have Sublime Text.
But it's not the first PR I've opened regarding typing on this repo, and the last time I was asked to add those

Reply:
Currently `mypy` probably accepts those, but if we run with `no_untyped_defs` it will yell at us.

Reply:
Thanks for the details. OK, if mypy is that needy, I guess we have to surrender and keep typing `__init__`